packages/radio/README.md

```diff
@@ -107,9 +109,12 @@ Standard radio buttons are the default style for radio buttons. They are optimal
         <sp-field-label for="example-2" size="l">
             <h4 class="spectrum-Heading--subtitle1">Invalid</h4>
         </sp-field-label>
-        <sp-radio-group id="example-2" name="example" vertical>
-            <sp-radio invalid value="kittens">Kittens</sp-radio>
-            <sp-radio invalid value="puppies" checked>Puppies</sp-radio>
+        <sp-radio-group invalid id="example-2" name="example" vertical>
+            <sp-radio value="kittens">Kittens</sp-radio>
+            <sp-radio value="puppies" checked>Puppies</sp-radio>
+             <sp-help-text slot="negative-help-text" icon>
+                This selection is invalid.
+            </sp-help-text>
```

Comment:
Here you've moved the `invalid` attribute off of the `<sp-radio>` elements. While it no longer seems to effect the elements visually, there is a functional effect to having this attribute: https://github.com/adobe/spectrum-web-components/blob/main/packages/radio/src/Radio.ts#L132-L138 We should confirm with the accessibility team the right path to achieving both the visual and screen reader specific changes you are targeting here.

Reply:
Great catch. Reaching out to accessibility and will update accordingly!

Reply:
Hey @Westbrook, so… from the accessibility team, invalid state on radio element is deprecated:

> In WAI-ARIA 1.2, the aria-invalid state is deprecated on role="radio" (see “inherited states and properties” under https://www.w3.org/TR/wai-aria-1.2/#radio), and I don’t think that screen readers will be consistent about announcing the invalid state when aria-invalid is specified on a input[type=radio], adding aria-invalid to the role="radiogroup" is the recommended approach, (see “supported states and properties” under https://www.w3.org/TR/wai-aria-1.2/#radiogroup).

But… this makes me wonder if we should doing a wider review for ARIA 1.2 deprecations? It looks like the Radio component itself does add `aria-invalid` in [Radio.ts](https://github.com/adobe/spectrum-web-components/blob/main/packages/radio/src/Radio.ts#L132)/`updated(changes:)`.

I'd suggest that if we do it would be a different ticket from this documentation update, but open to discussing that more.

Reply:
Changing functionality in a separate PR (please create an issue to keep this filed) is a great idea.

I'd say the middle ground is to keep the `invalid` attribute on the `<sp-radio>` elements for this PR, so that the _intended_ outcome is achieve regardless of whether it's the right one for now.

Reply:
Issue created: https://github.com/adobe/spectrum-web-components/issues/3571